lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -141,8 +154,9 @@ var WebMapServiceCatalogItem = function(terria) {
                 return undefined;
             }
             var layer = this.layers.split(',')[0];
+            // This is a double fallback. Generally we get a legend URL from the getCapabilities, or it's not supplied in the init file.
             return new LegendUrl(cleanUrl(this.url) +
-                '?service=WMS&version=1.3.0&request=GetLegendGraphic&format=image/png&layer=' + layer, 'image/png');
+                '?service=WMS&version=1.1.0&request=GetLegendGraphic&format=image/png&transparent=True&layer=' + layer, 'image/png');
```

Comment:
When this is a geoserver, shouldn't this URL include the font size and antialiasing tweaks too?  Otherwise the Mobile Black Spot Programme datasets don't get a nice legend because they don't have an explicit style-with-legend listed in GetCapabilities.